### DISCUSSION TITLE
Train DBNet_r18 model in CTW1500 data set, the effect is relatively poor, can anyone help me see, is there any problem?

### DISCUSSION THREAD

**Discussion by qiyueliuhuo** [open-mmlab/mmocr#1641] [Q&A]
I use dbnet_resnet18_fpnc_100k_synthtext-2e9bf392.pth  as pretrain model
The log file is as follows
[20221221_222607.log](https://github.com/open-mmlab/mmocr/files/10294002/20221221_222607.log)

When training to 180 EPOCH, the model performance is the best. It seems that subsequent training is invalid. Why is this?

**Comment by qiyueliuhuo** [comment]
The result in the original paper is  82.4 76.6 79.4，My training results are 0.7635  0.6761  0.7171

**Reply by gaotongxiao** [reply]
We haven't trained DB on CTW yet, but one thing that I'm sure of is that the test-time image size should be different from IC15. While the authors didn't release their configuration for CTW, you could refer to other models' CTW configurations in MMOCR and try them on DB.

**Reply by qiyueliuhuo** [reply]
Thank you for your reply. There are indeed some post-processing parameters and test-image size are different from IC15. I am still trying...
